### ISSUE TITLE
Replicating PPO Paper A2C results

### ISSUE THREAD

**Issue by Breakend** [openai/baselines#125]
I'm going through and trying to replicate everything from the PPO paper.

@joschu Is there some insight into what parameters/codebase/etc. was used for the A2C work in the paper? I spent a little bit of time trying to modify the Atari A2c code for MuJoCo tasks, but then I thought it might be worth it to ask if there are any plans to release this code in the near future as that would be a more fair effort to replicate the results from the codebase directly (I'm getting pretty poor results with my modifications so i'm wondering if I missed something).

Thanks!

**Comment by joschu** [comment]
The released code should replicate the Atari results.
The A2C MuJoCo results are from some branch that's not prepared for release, and I don't have time to dig it up.

**Comment by johnholl** [comment]
My experience running this A2C code with its default parameters has been performance considerably below those reported in the graphs here:

[https://blog.openai.com/baselines-acktr-a2c/](url)

Namely, the large grid of graphs that show a2c in red. I've had trouble getting good results on Riverraid, Alien, and Chopper command.

Can anyone clarify if hyperparameters in these graphs are the same as the default hyperparameters in the a2c code (ie 16 workers, 5 step time unrolls, loss function coefficients, learning rate/optimizer)?

**Comment by joschu** [comment]
The A2C code in baselines is the same as the code that generated the results in the paper. It's possible that there was a tiny change to image processing or something, or maybe it's just the random seed.

**Comment by LinZichuan** [comment]
Hi, does anyone find A2C code that can reproduce PPO paper results?

**Comment by NishanthVAnand** [comment]
I am still now able to find the hyperparameters to be used for a2c on Mujoco tasks? Can someone help me with that?
